version.json

```diff
@@ -1,3 +1,3 @@
 {
-  "version": "v0.8.0-rc2"
```

Comment:
I haven't updated the example project's dependency in this PR. Not sure if that's something to do now or post a release.

Reply:
Are there reasons for not to?  I can think of reasons to update, such as running tests, but not for not updating.

Reply:
Wasn't sure what to update to. WDYT commit or the tag number? Either way probably let's merge this now and we'll document this for the next one (trying to get kubo and other PRs merged and don't want to wait for more CI runs at the moment)

Reply:
sgtm